tests/e2e/e2e_ingressbackend_test.go

```diff
@@ -33,7 +37,10 @@ var _ = OSMDescribe("Ingress using IngressBackend API",
 	})
 
 func testIngressBackend() {
-	var destNs = framework.RandomNameWithPrefix("server")
+	var (
+		destNs     = framework.RandomNameWithPrefix("server")
+		secretName = "ingress-gateway-cert"
+	)
 
 	It("allows ingress traffic", func() {
```

Comment:
Could the changes made to the IngressBackend test be added to a separate "It" block? We could have "allows ingress traffic without TLS" and "allows ingress traffic with mTLS"? This would allow us to test our support for both documented scenarios https://main--osm-docs.netlify.app/docs/demos/ingress_k8s_nginx/